tools/AccelSearch.py

```diff
@@ -48,7 +48,7 @@ def AccelSearch(filename):
                 logcount += 1
             return False
         if m.get_type() == 'PARAM_VALUE':
-            if m.param_id.startswith('INS_PRODUCT_ID'):
+            if m.param_id.decode('ascii').startswith('INS_PRODUCT_ID'):
```

Comment:
Here I would do `m.param_id.startswith(b'INS_PRODUCT_ID')`. But semantically it is the same.